src/applications/personalization/profile360/selectors.js

```diff
@@ -26,7 +26,8 @@ export const directDepositAddressIsSetUp = state => {
 };
 
 export const directDepositIsBlocked = state => {
-  const controlInfo =
-    directDepositInformation(state)?.responses?.[0]?.controlInformation || {};
+  const controlInfo = directDepositInformation(state)?.responses?.[0]
```

Comment:
I think that this function is a little ugly internally, but works with the unit tests which are solid. But I'm open to other ideas to make this a little more elegant.